gtest/src/program.rs

```diff
@@ -79,6 +79,10 @@ pub struct Program<'a> {
 }
 
 impl<'a> Program<'a> {
+    pub fn calculate_program_id(code_hash: CodeHash, salt: &[u8]) -> ProgramId {
+        ProgramId::generate(code_hash, salt)
+    }
+
```

Comment:
I don't like this method (especially inside the `Program` impl) due to ability to manually set id of the program in `gtest`.

Reply:
discussed, will be moved as a separate function

Reply:
Isn't it?

Reply:
moved in [2045fb6](https://github.com/gear-tech/gear/pull/755/commits/2045fb671b8ff915991b79b3af1874092e28c3d3)